docs/content/policy-reference.md

```diff
@@ -931,6 +931,7 @@ The table below shows examples of calling `http.send`:
 | Accessing Google using System Cert Pool | ``http.send({"method": "get", "url": "https://www.google.com", "tls_use_system_certs": true })`` |
 | Files containing TLS material | ``http.send({"method": "get", "url": "https://127.0.0.1:65331", "tls_ca_cert_file": "testdata/ca.pem", "tls_client_cert_file": "testdata/client-cert.pem", "tls_client_key_file": "testdata/client-key.pem"})`` |
 | Environment variables containing TLS material | ``http.send({"method": "get", "url": "https://127.0.0.1:65360", "tls_ca_cert_env_variable": "CLIENT_CA_ENV", "tls_client_cert_env_variable": "CLIENT_CERT_ENV", "tls_client_key_env_variable": "CLIENT_KEY_ENV"})`` |
+| Unix Socket URL Format| ``` http.send({"method": "get", "url": "unix://localhost/?socket=%F2path%F2file.socket"})``` |
```

Comment:
[nit] The other examples use double-backticks, not triple. I think there's a difference 😅